### ISSUE TITLE
az aks browse fails even when network connection is stable

### ISSUE THREAD

**Issue by hostettler** [Azure/azure-cli#10214]
**Command Name**
`az aks browse`

**Errors:**
```
HTTPConnectionPool(host='localhost', port=8888): Max retries exceeded with url: /closeport/8001 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x04F57990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',))
Traceback (most recent call last):
pip-install-v0p8dbpk\urllib3\urllib3\connection.py, ln 159, in _new_conn
urllib3\urllib3\util\connection.py, ln 80, in create_connection
urllib3\urllib3\util\connection.py, ln 70, in create_connection
...
pip-install-wl9gly17\requests\requests\adapters.py, ln 516, in send
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8888): Max retries exceeded with url: /closeport/8001 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x04F57990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',))
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks browse --resource-group {} --name {}`

## Environment Summary
```
Windows-10-10.0.17763-SP0
Python 3.6.6
Shell: cmd.exe

azure-cli 2.0.67 *

```
## Additional Context
Network was stable and fast

**Comment by seanmck** [comment]
This specific error was covered in the fix for #7686. We still need to handle the idle timeout connection drops, which is covered by #11580.

**Comment by seanmck** [comment]
Duplicate of #11580
